js/pages/data-sources/data-sources.less

```diff
@@ -1,4 +1,8 @@
 .data-sources {
+	& .drilldown-enabled tr,rect {
```

Comment:
these rules will result in the following css:
```
.data-sources .drilldown-enabled tr,
.data-sources rect
```
is that what you're trying to achieve?

Reply:
No, was trying to achieve:

.data-sources .drilldown-enabled tr,
.data-sources .drilldown-enabled rect

Can you suggest the proper rules?

Reply:
I think this:

```
.data-sources {
	&.drilldown-enabled {
    	tr, rect {
      font-size: 1px // add styles here
		}
  	}
}
```

Validated here: http://winless.org/online-less-compiler

Reply:
Fixed in https://github.com/OHDSI/Atlas/pull/1165/commits/c5d33800a2c54993a6ce045d8e5a375270fc1d58